src/test/java/fr/quatrevieux/araknemu/game/player/characteristic/PlayerLifeTest.java

```diff
@@ -181,4 +181,22 @@ void setWillDispatchEvent() throws SQLException, ContainerException {
         assertEquals(123, life.current());
         assertEquals(123, ref.get().current());
     }
+
+    @Test
+    void lifeRegenerationIsCorrect() throws Exception{
```

Comment:
Some tests are missing :
- current() during regen
- setLifeWithCurrentRegeneration()

The two tests are too similar : change sleep duration and regen speed to get higher difference between two.